app/services/current-session.js

```diff
@@ -48,4 +51,12 @@ export default class CurrentSessionService extends Service {
       });
     }
   }
+
+  get isAdmin() {
+    let roles = this.roles;
+    if (this.impersonation.isImpersonating) {
+      roles = this.impersonation.originalRoles || [];
+    }
+    return roles.includes(ENV.adminRole);
+  }
```

Comment:
Nitpick but I'm not sure if we really need the env variable here, creates extra overhead. See pr of subsidiepunt which skipped that https://github.com/lblod/frontend-subsidiepunt/pull/40/files#diff-e2ecd07ef6a6246eb0af68ff590449a30c991409ca69992ee64d9a878888e4c8R20-R26

Reply:
What do you mean with creating extra overhead? Do you mean mental overhead of having to look for the variable in another file?

Reply:
I guess it doesn't really need to be configurable since we probably know the admin role already. Less configuration in the backend app that way as well.

Reply:
> What do you mean with creating extra overhead? Do you mean mental overhead of having to look for the variable in another file?

I meant for the local setup you have to set the env variable

Reply:
There is already a couple of variables that need to be overridden in that config. I don't why 1 extra would create overhead. At least it is in an easy to reach location for if this role needs to be adjusted. (And it probably will need to be, because I have no way to check that I have the correct value.) I will hard code it in the config as a compromise.

Reply:
It just feels silly to make it configurable when we do know the role, or at least we should be able to get it. The dashboard only did it this way because that's a generic frontend.

Anyways, if you feel strongly about keeping it flexible, fair enough :smile:. I agree it does not matter that much.

> because I have no way to check that I have the correct value.

Why not? You get the list of roles when you log in, and Nele should also be able to give you the correct one.